code/_globalvars/bitfields.dm

```diff
@@ -152,15 +152,14 @@ DEFINE_BITFIELD(interaction_flags_atom, list(
 ))
 
 DEFINE_BITFIELD(interaction_flags_machine, list(
-	"INTERACT_MACHINE_ALLOW_SILICON" = INTERACT_MACHINE_ALLOW_SILICON,
-	"INTERACT_MACHINE_OFFLINE" = INTERACT_MACHINE_OFFLINE,
 	"INTERACT_MACHINE_OPEN" = INTERACT_MACHINE_OPEN,
+	"INTERACT_MACHINE_OFFLINE" = INTERACT_MACHINE_OFFLINE,
+	"INTERACT_MACHINE_WIRES_IF_OPEN" = INTERACT_MACHINE_WIRES_IF_OPEN,
+	"INTERACT_MACHINE_ALLOW_SILICON" = INTERACT_MACHINE_ALLOW_SILICON,
 	"INTERACT_MACHINE_OPEN_SILICON" = INTERACT_MACHINE_OPEN_SILICON,
+	"INTERACT_MACHINE_REQUIRES_SILICON" = INTERACT_MACHINE_REQUIRES_SILICON,
 	"INTERACT_MACHINE_REQUIRES_SIGHT" = INTERACT_MACHINE_REQUIRES_SIGHT,
 	"INTERACT_MACHINE_REQUIRES_LITERACY" = INTERACT_MACHINE_REQUIRES_LITERACY,
-	"INTERACT_MACHINE_REQUIRES_SILICON" = INTERACT_MACHINE_REQUIRES_SILICON,
-	"INTERACT_MACHINE_SET_MACHINE" = INTERACT_MACHINE_SET_MACHINE,
-	"INTERACT_MACHINE_WIRES_IF_OPEN" = INTERACT_MACHINE_WIRES_IF_OPEN,
```

Comment:
This isn't alphabetized but I've found it way easier when it's set in the order in which they are defined so I can easily know when one is missing.